app/models/distribution.rb

```diff
@@ -71,6 +73,10 @@ class Distribution < ApplicationRecord
     where("issued_at > :start_date AND issued_at <= :end_date",
           start_date: Time.zone.today.beginning_of_week.beginning_of_day, end_date: Time.zone.today.end_of_week.end_of_day)
   end
+  scope :in_last_12_months, -> do
+    where("issued_at > :start_date AND issued_at <= :end_date",
```

Comment:
I want to highlight that I'm using `x.months.ago` style as the `start_date` and `Time.zone. ...` style in `end_date`. 💭 I’m wondering if this approach is ideal and whether it might create any conflicts with time zones. 

I tested locally using both `12.months.ago.beginning_of_day` and `Time.zone.today.beginning_of_day - 12.months` and it seems to be working fine. However, I’d appreciate your advice on whether this is a reliable approach. 🙏

Reply:
`ago` uses the `Time.zone` approach - you should be good.

```
[2] pry(main)> 1.month.ago.class
=> ActiveSupport::TimeWithZone
```